MyMovies/View Controllers/MovieTableViewCell.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MovieTableViewCell.swift
+//  MyMovies
+//
+//  Created by Jarren Campos on 5/1/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class MovieTableViewCell: UITableViewCell {
+    
+    var movie: Movie? {
+        didSet {
+            updateViews()
+        }
+    }
+    var movieContoller: MovieController?
+    
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    @IBOutlet weak var hasWatchedButton: UIButton!
+    
+    @IBAction func hasWatchedButtonTapped(_ sender: Any) {
+        movie?.hasWatched.toggle()
+        do {
+            try CoreDataStack.shared.save()
+            if let movie = movie {
+                movieContoller?.sendMovieToFirebase(movie: movie)
+                
+            }
+        } catch {
+            NSLog("Error saving hasWatched")
+        }
+        
```

Comment:
Make sure to clean up these whitespaces before you do your last commit and push.